app/views/enrollments/_form.html.haml

```diff
@@ -2,8 +2,9 @@
   = f.error_notification
   = f.error_notification message: f.object.errors[:base].to_sentence if f.object.errors[:base].present?
   .form-inputs
-    = f.association :survey
-    = f.association :user
-    = f.input :status
-  .form-actions
+    = f.association :survey, collection: Survey.order(:survey_name), prompt: "Choose a Survey", label_method: :survey_name 
+    = f.association :user, collection: User.where.not(first_name: nil).order(:first_name), prompt: "Choose a Name", label_method: :full_name 
+    = f.input :status, collection: Enrollment.statuses.keys.map {|name| [name.titleize, name]}, prompt: "Status"
+  .form-actions   
```

Comment:
Fixed the select menus to populate the survey & user names & status

Reply:
![image](https://user-images.githubusercontent.com/78874252/190228754-d9eea06d-9804-4b14-bf82-050a7cc4a07d.png)